### PR TITLE
MAINT: Bump to v0.6.0

### PR DESCRIPTION
**Pull request by HaoZeke**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #36

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - pip
+    - setuptools >=61.2
+    - setuptools_scm[toml] >=6.2
+    - wheel
```
Comment (xhochy) [inline recipe/meta.yaml]: This should be in host 

Also [] is not supported syntax.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
-    - six
   run:
     - python
-    - setuptools
-    - six
+    - pip
+    - asv_runner
+    - json5
+    - tabulate
+    - colorama  # [os_name == 'nt']
```
Comment (xhochy) [inline recipe/meta.yaml]: Please use [win]

**Review by xhochy — DISMISSED** [review]

**Comment by HaoZeke** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 3]
```diff
@@ -22,14 +22,17 @@ requirements:
...
   run:
     - python
-    - setuptools
-    - six
+    - asv_runner
+    - json5
+    - tabulate
+    - colorama  # [win]
+    - pyyaml  # [platform_python_implementation != "PyPy"]
+    - pympler  # [platform_python_implementation != "PyPy"]
```
Comment (h-vetinari) [inline recipe/meta.yaml]:
```suggestion
    - asv_runner >=0.0.9
    - json5
    - tabulate
    - colorama  # [win]
    - pyyaml    # [python_impl != "pypy"]
    - pympler   # [python_impl != "pypy"]
```

**Review by h-vetinari — APPROVED** [review]
Modulo a nit; let's reflect the lower bound for `asv_runner` from [upstream](https://github.com/airspeed-velocity/asv/blob/v0.6.0/pyproject.toml#L28).

**Comment by adityagoel4512** [comment]
Not much experience with conda-forge here but are you supposed to have asv-runner in the run section? It isn't on conda-forge.

**Comment by xhochy** [comment]
> Not much experience with conda-forge here but are you supposed to have asv-runner in the run section? It isn't on conda-forge.

It is on conda-forge: https://github.com/conda-forge/asv_runner-feedstock

**Comment by adityagoel4512** [comment]
Oops, thank you.

**Comment by adityagoel4512** [comment]
> > Not much experience with conda-forge here but are you supposed to have asv-runner in the run section? It isn't on conda-forge.
> 
> It is on conda-forge: https://github.com/conda-forge/asv_runner-feedstock

Ah I see, I was searching for asv-runner.

**Pull request merged**